tools/stack_analysis.sh

```diff
@@ -4,13 +4,28 @@
 # size information, and is thus most easily called using the `make stack-analysis`
 # rule.
 
-# Print the stack frame size of `main`
-printf "main stack frame: \n"
-$(find $(rustc --print sysroot) -name llvm-readobj) --elf-output-style GNU --stack-sizes $1 | grep 'main'
+bold=$(tput bold)
+normal=$(tput sgr0)
```

Comment:
Just a heads-up this adds a dependency on `ncurses`. It's shipped with my distro and seems to be on many, but it's worth keeping in mind - for instance I'm not quite sure whether Cygwin would have it by default.

Reply:
We've had `tput` around in the tools and build system for a long time; not a new dependency here, so I think it's fine.